state/azure/cosmosdb/storedprocedure.go

```diff
@@ -116,3 +118,8 @@ function dapr_multi_v2(operations) {
         }
     }
 }`
+
+const spVersionDefinition = `function daprSpVersion(prefix) {
```

Comment:
This is the new sproc. It just returns a constant with the version of the stored procedure